docs/package_layout.md

```diff
@@ -15,3 +15,19 @@ should be in the plugin directory and will be automatically loaded at runtime.
 ## Runtime section
 The runtime part contains information which will be used only at runtime
 to find files during the checking process.
+
+### Analyzers section
```

Comment:
I think we should sync this section with the main README [configuring-clang-version](https://github.com/Ericsson/codechecker/blob/master/docs/README.md#configuring-clang-version)
We could mention the environment variable there too.